src/client/theme-default/components/VPButton.vue

```diff
@@ -35,13 +39,35 @@ const component = computed(() => {
     :target="props.target ?? (isExternal ? '_blank' : undefined)"
     :rel="props.rel ?? (isExternal ? 'noreferrer' : undefined)"
   >
-    {{ text }}
+    <div class="start-icon" v-if="startIcon">
```

Comment:
div is not allowed inside button

Reply:
Got it, i try to use <span> instead of <div>, would it still break ssr?

Reply:
Got it. I try to use `span` instead of `div`, would it still break ssr?

Reply:
no, span should work. (only phrasing content is allowed inside buttons - https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#phrasing_content)

Reply:
Thanks, I see. 😄